src/qml/main.qml

```diff
@@ -141,6 +142,55 @@ Application {
                     iconName: "ios-bluetooth-outline"
                     onClicked: layerStack.push(bluetoothLayer)
                 }
+                ListItem {
+                    //% "WiFi"
+                    title: qsTrId("id-wifi-page")
+                    iconName: "ios-wifi-outline"
+                    onClicked: layerStack.push(wifiLayer)
+                }
```

Comment:
I think we should guard this entry with a visible clause similar to the one for "Sound" so that porters can specify whether Wi-Fi is supported by the port and we only show this menu when it's relevant and not a misleading broken UX on watches where it's not supported by the port

Reply:
yep, that sounds like a good idea